docs/source/en/tasks/semantic_segmentation.md

```diff
@@ -245,7 +245,7 @@ logits first, and then reshaped to match the size of the labels before you can c
 ...             reduce_labels=False,
 ...         )
 ...         for key, value in metrics.items():
-...             if type(value) is np.ndarray:
+...             if isinstance(value, np.ndarray)
```

Comment:
I feel a bit strange here `:` is removed. Is this expected?

Reply:
nice catch! Thanks